assets/js/Ioda/components/table/RawSignalsTable.js

```diff
@@ -68,7 +68,7 @@ const RawSignalsTable = ({
   data,
   entityType,
   toggleEntityVisibilityInHtsViz,
-  handleEntityClick,
+  // handleEntityClick,
```

Comment:
Calling `handleEntityClick` throws an exception in prod when clicking on an item in the table. After upgrading, the links stop working due to the exceptions (@amandahmeng discovered this). I have commented them out here and will clean it up if everything is ok. See the video attached for the behavior in production.

https://github.com/InetIntel/ioda-ui/assets/15727660/4536eb0a-d3d1-4e65-8f7d-c6199ff41f99



cc @salcock